README.md

```diff
@@ -47,6 +47,14 @@ npm install watson-developer-cloud
 ```
 
 ## Getting the service credentials
+
+## Visual Recognition
+
+The process for authenticating with Visual Recognition has changed:
+
+- For new service instances, authenticate by using IAM. See [IAM Authentication](#iam-authentication). Also set the service URL by passing in `url` in the service constructor. 
+- For service instances created before May 23, 2018, authenticate by providing the API Key for the service instance.
```

Comment:
because you don't have an API key section, perhaps modify to 
```markdown
- For service instances created before May 23, 2018, authenticate by providing the basic auth `api_key` for the service instance.
```